src/test/java/lotto/LottoNumbersTest.java

```diff
@@ -0,0 +1,34 @@
+package lotto;
+
+import lotto.model.BonusBall;
+import lotto.model.LottoNumbers;
+import lotto.model.MatchCount;
+import lotto.model.WinningNumbers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import java.util.Arrays;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class LottoNumbersTest {
+
+    @Test
+    @DisplayName("로또 번호가 6개가 아니면 예외를 발생")
+    void throwExceptionIfLottoNumbersAreNotSix() {
+        assertThrows(IllegalArgumentException.class, () ->
+            new LottoNumbers(Arrays.asList(1, 2, 3, 4, 5))
+        );
```

Comment:
JUnit의 Assertion을 사용해주셨네요!

지난 미션에서 배우셨듯, 검증은 JUnit의 메서드를 사용하지 않고, AssertJ의 메서드를 주로 사용해요.

```suggestion
        assertThatThrownBy(() -> new LottoNumbers(Arrays.asList(1, 2, 3, 4, 5)))
                .isInstanceOf(IllegalArgumentException.class);
```